source/ranges.tex

```diff
@@ -3802,1398 +3855,2476 @@
+\begin{itemdecl}
+constexpr default_sentinel_t end() const;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects Equivalent to: \tcode{return default_sentinel;}
+\end{itemdescr}
+
+\rSec3[range.split.inner]{Class template \tcode{split_view::inner_iterator}}
+
+\begin{codeblock}
+namespace std::ranges {
+  template<class V, class Pattern>
+  template<bool Const>
+  struct split_view<V, Pattern>::inner_iterator {
+  private:
+    using Base =
+      conditional_t<Const, const V, V>;                 // \expos
+    outer_iterator<Const> i_ = outer_iterator<Const>(); // \expos
+    bool incremented_ = false;                          // \expos
+  public:
+    using iterator_concept  = typename outer_iterator<Const>::iterator_concept;
+    using iterator_category = @\seebelow@;
+    using value_type        = range_value_t<Base>;
+    using difference_type   = range_difference_t<Base>;
+
+    inner_iterator() = default;
+    constexpr explicit inner_iterator(outer_iterator<Const> i);
+
+    constexpr decltype(auto) operator*() const { return *i_.@\placeholder{current}@; }
+
+    constexpr inner_iterator& operator++();
+    constexpr decltype(auto) operator++(int) {
+      if constexpr (ForwardRange<V>) {
+        auto tmp = *this;
+        ++*this;
+        return tmp;
+      } else
+        ++*this;
+    }
+
+    friend constexpr bool operator==(const inner_iterator& x, const inner_iterator& y)
+      requires ForwardRange<Base>;
+    friend constexpr bool operator!=(const inner_iterator& x, const inner_iterator& y)
+      requires ForwardRange<Base>;
+
+    friend constexpr bool operator==(const inner_iterator& x, default_sentinel_t);
+    friend constexpr bool operator==(default_sentinel_t, const inner_iterator& x);
+    friend constexpr bool operator!=(const inner_iterator& x, default_sentinel_t y);
+    friend constexpr bool operator!=(default_sentinel_t y, const inner_iterator& x);
+
+    friend constexpr decltype(auto) iter_move(const inner_iterator& i)
+    noexcept(noexcept(ranges::iter_move(i.i_.@\placeholdernc{current}@))) {
+      return ranges::iter_move(i.i_.@\placeholdernc{current}@);
+    }
+
+    friend constexpr void iter_swap(const inner_iterator& x, const inner_iterator& y)
+      noexcept(noexcept(ranges::iter_swap(x.i_.@\placeholdernc{current}, y.i_.\placeholdernc{current}@)))
+      requires IndirectlySwappable<iterator_t<Base>>;
+  };
+}
+\end{codeblock}
+
+\pnum
+The \grammarterm{typedef-name} \tcode{iterator_category} denotes
+\tcode{forward_iterator_tag} if
+\tcode{iterator_traits<iterator_t<Base>>::iterator_category} models
+\tcode{DerivedFrom<forward_iterator_tag>}, and \tcode{input_iterator_tag}
+otherwise.
+
+\indexlibrary{\idxcode{inner_iterator}!\idxcode{split_view::inner_iterator}}%
+\begin{itemdecl}
+constexpr explicit inner_iterator(outer_iterator<Const> i);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects Initializes \tcode{i_} with \tcode{i}.
+\end{itemdescr}
+
+\indexlibr
```

Comment:
Remove space between `iterator` and `&`.
```suggestion
    iterator& operator=(const iterator&) = delete;
```

Reply:
Fixed.